account_payment_return_financial_risk/tests/test_payment_return_risk.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2017-2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestPartnerPaymentReturnRisk(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestPartnerPaymentReturnRisk, cls).setUpClass()
+        cls.journal = cls.env['account.journal'].create({
+            'name': 'Test Sales Journal',
+            'code': 'tVEN',
+            'type': 'sale',
+            'update_posted': True,
+        })
+        cls.bank_journal = cls.env['account.journal'].create({
+            'name': 'Test Bank Journal',
+            'code': 'BANK',
+            'type': 'bank',
+            'update_posted': True,
+        })
+        cls.account_type = cls.env['account.account.type'].create({
+            'name': 'Test',
+            'type': 'receivable',
+        })
+        cls.account = cls.env['account.account'].create({
+            'name': 'Test account',
+            'code': 'TEST',
+            'user_type_id': cls.account_type.id,
+            'reconcile': True,
+        })
+        cls.account_income = cls.env['account.account'].create({
+            'name': 'Test income account',
+            'code': 'INCOME',
+            'user_type_id': cls.env['account.account.type'].create(
+                {'name': 'Test income'}).id,
+        })
+        cls.partner = cls.env['res.partner'].create({'name': 'Test'})
+        cls.invoice = cls.env['account.invoice'].create({
+            'journal_id': cls.journal.id,
+            'account_id': cls.account.id,
+            'company_id': cls.env.user.company_id.id,
+            'currency_id': cls.env.user.company_id.currency_id.id,
+            'partner_id': cls.partner.id,
+            'invoice_line_ids': [(0, 0, {
+                'account_id': cls.account_income.id,
+                'name': 'Test line',
+                'price_unit': 50,
+                'quantity': 10,
+            })]
+        })
+        cls.reason = cls.env['payment.return.reason'].create({
+            'code': 'RTEST',
+            'name': 'Reason Test'
+        })
+        cls.invoice.signal_workflow('invoice_open')
```

Comment:
You must change `cls.invoice.signal_workflow('invoice_open')` by `cls.invoice.action_invoice_open()`